product_docs/docs/pgd/5/durability/limitations.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Limitations
+---
+
+The following limitations apply to the use of commit scopes and the various durability options they enable.
+
+
+Group Commit transactions can't yet execute DDL, and they don't support [explicit
+two-phase commit](../twophase). Future releases might allow them. However, the `TRUNCATE`
+command is allowed.
+
+You can combine only CAMO transactions with the `DEGRADE TO` clause for
+switching to asynchronous operation in case of lowered availability.
+
+Group Commit, Eager and CAMO transactions aren't currently supported in
+combination with the Decoding Worker feature or with transaction streaming.
+Installations using these commit scopes must keep `enable_wal_decoder` and `streaming_mode`
+disabled for the PGD node group.
+
+[Legacy synchronous replication](legacy-sync) uses a mechanism for transaction
+confirmation different from Group Commit. The two aren't compatible, and you
+must not use them together. Therefore, whenever you Group Commit transactions,
+make sure none of the PGD nodes are configured in `synchronous_standby_names`.
```

Comment:
Probably a question for @PJMODOS or @mwanner - could we technically prevent this by raising an error if the user enables legacy synchronous replication?